cmd/ak/cmd/vars/set.go

```diff
@@ -32,16 +32,35 @@ var setCmd = common.StandardCommand(&cobra.Command{
 		}
 
 		var value string
-		if len(args) == 2 {
-			value = args[1]
+
+		isValueSet := len(args) > 1
```

Comment:
isValueProvided/Supplied?
because now it's not clear whether we are talking about old value from DB or value supplied by the user

Reply:
Or just don't use a var. `len(args) > 1` is trivial and obvious, the var isn't really adding self-documentation.